MIPs/mip-1.md

```diff
@@ -3,7 +3,7 @@ MIP: 1
 Title: Extend `wallet_watchAsset` to Support ERC-721 and ERC-1155 Tokens (NFTs)
 Status: Implemented
 Stability: Experimental
-discussions-to: https://github.com/MetaMask/metamask-improvement-proposals/discussions
+discussions-to: https://github.com/MetaMask/metamask-improvement-proposals/discussions/22
```

Comment:
Can we pull these redirect changes into a different PR?

Reply:
Yeah, I probably should have done that.